source/index.html

```diff
@@ -1,12 +1,152 @@
+            <h2 class="reasons__title">Сувениры</h2>
+            <p class="reasons__text">Не только китайского, но и местного производства!</p>
+          </li>
+          <li class="reasons__item reason--icon-food">
+            <h2 class="reasons__title">Еда</h2>
+            <p class="reasons__text">Всегда заказывайте фирменный бургер, вы не разочаруетесь!</p>
+          </li>
+        </ul>
+      </div>
+
+      <div class="reasons__container">
+        <div class="reason reason__color--blue">
+          <h2 class="reasons__title">Там есть мост дьявола</h2>
+          <div class="reason reason__digit">№2</div>
+          <p class="reasons__text">Да, по нему можно пройти! если вы осмелитесь, конечно</p>
+        </div>
+      </div>
+
+      <div class="reason-picture">
+        <img src="" alt="" width="" height="">
+      </div>
+
+      <div class="reasons__container">
+        <div class="reason reason__color--gray">
+          <h2 class="reasons__title">Небольшая площадь</h2>
+          <div class="reason reason__digit">№3</div>
+          <p class="reasons__text">Все интересные места находятся очень близко</p>
+        </div>
+        <div class="reason reason__color--gray">
+          <h2 class="reasons__title">Красивая дорога</h2>
+          <div class="reason reason__digit">№4</div>
+          <p class="reasons__text">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+        </div>
+        <div class="reason reason__color--gray">
+          <h2 class="reasons__title">Мало туристов</h2>
+          <div class="reason reason__digit">№5</div>
+          <p class="reasons__text">Большинство едет в гранд каньон и толпится там</p>
+        </div>
+      </div>
+    </article>
+
+    <div class="search-hotel">
+      <h2 class="search__title">Заинтересовались?</h2>
+      <p class="search__description">Укажите продполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в
+        Седоне</p>
+      <button type="button" class="button--search">Найти гостиницу</button>
+    </div>
+
+    <div class="address__map">Карта</div>
+
+    <footer class="footer">
+      <div class="footer__container">
+        <a href="#" class="hashtag">#visitsedona</a>
+      </div>
+      <div class="footer__social social">
+        <ul class="social__list">
+          <li class="socail__item">
+            <a class="social__link social__link--twitter" href="#">Твиттер</a>
+          </li>
+          <li class="socail__item">
+            <a class="social__link social__link--facebook" href="#">Фейсбук</a>
+          </li>
+          <li class="socail__item">
+            <a class="social__link social__link--youtube" href="#">Ютуб</a>
+          </li>
+        </ul>
+      </div>
+      <div class="footer__copyright copyright">
+        <b class="copyright__text">Разработано</b>
+        <a class="copyright__logo" href="#">
+          <img src="../source/img/htmlacademyfooter.svg" alt="" width="27" height="34">
```

Comment:
Тоже избыточный адрес и нет alt
```suggestion
          <img src="./img/htmlacademyfooter.svg" alt="" width="27" height="34">
```